qa-tests-backend/scripts/run-compatibility.sh

```diff
@@ -13,7 +13,7 @@ source "$ROOT/qa-tests-backend/scripts/lib.sh"
 set -euo pipefail
 
 compatibility_test() {
```

Comment:
whatabout doing this all in bash? in that way it'll reuse the same wrapping cluster. Which will save a bit of time.